### PR TITLE
Added Elif and Elif2

### PR DESCRIPTION
**Pull request by sofiabrown91**

### CODE REVIEW

File: src/Elif2.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+public class Elif2 {
+
+    // BAhar Elif
+    //
+    //
```
Comment (NailBul) [inline src/Elif2.java:5]: No need for this line

File: src/Elif.java [hunk 2]
```diff
@@ -4,4 +4,6 @@ public class Elif {
 
     //line2
 
+    //line3
```
Comment (NailBul) [inline src/Elif.java:7]: Check it please

Comment (sofiabrown91) [reply src/Elif.java:7]: ok thank you

Comment (NailBul) [reply src/Elif.java:7]: good to go

**Review by NailBul — APPROVED** [review]

**Pull request merged**